contrib/drivers/lvm/lvm.go

```diff
@@ -202,13 +207,65 @@ func (d *Driver) lvHasSnapshot(lvPath string) bool {
 	args := []string{"--noheading", "-C", "-o", "Attr", lvPath}
 	info, err := d.handler("lvdisplay", args)
 	if err != nil {
-		log.Error("Failed to remove logic volume:", err)
+		log.Error("Failed to display logic volume:", err)
 		return false
 	}
 	info = strings.Trim(info, " ")
 	return info[0] == 'o' || info[0] == 'O'
 }
 
+func (d *Driver) getActiveSnapshotsPathsOfLv(lvPath string) ([]string, error) {
+	array := strings.Split(lvPath, "/")
+	lvName := array[len(array)-1]
+	var snapshotsPaths []string
+
+	args := []string{}
+	info, err := d.handler("lvdisplay", args)
```

Comment:
It would be much more concise to use use "lvdisplay --noheading -C -o Attr" instead of the full default human readable output of lvdisplay. That would give

Reply:
If I use "lvdisplay --noheading -C -o Attr", I can't get the value of "LV snapshot status".

Reply:
OK, thanks for pointing that out. It seems like there should be a better way to match up snapshot volumes with their parents, but I don't know off the top of my head.

One issue with this though - I don't believe it will work with thin provisioned volumes. It looks like in that case it is not "LV snapshot status" but rather "LV Thin origin name  <name>"

Reply:
Because my ultimate goal is to deactivate the snapshot with the status "active", so I use "LV snapshot status" instead of "LV Thin origin name".